.github/PULL_REQUEST_TEMPLATE/release-pull-request-template.md

```diff
@@ -0,0 +1,38 @@
+# Stravalib Release Pull Request Template
```

Comment:
Ok no worries... so i'm super glad you asked this because it actually works a bit differently than i thought it worked. 

You can make [issue templates ](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)which are nice because they allow someone submitting a new issue to chose the type of issue. You can even have github automatically label issues by type. and in the template you ask people for specific information like for us we may want to see the users environment list, version of python, etc for all bugs reported. 

Pull request templates are a little less nice. We can have a nice default template (which i just added i'll ping you on the file). that text will then appear on any new PR opened. we can ask users to ensure they read the contributing guide for instance, and opened an issue first. I asked them to add closes #issue-number-here at the top because with that text GitHub will close their issue when we merge making maintenance a bit easier! 

Now the release template let's see if we like this. It turns out that we need to craft a specific URL to grab that release PR template. IT will just be you and I making releases i think for the most part. So let's see if we like this. I added a LINK at the bottom of the normal PR template to

It will look like this:
(i added release-1.1 as the branch name you'd have to edit the url to make sure the branch is right). I haven't tested 
https://github.com/stravalib/stravalib/compare/master...release-1.1?template=release-pull-request-template.md

I think i've set this up so you can see how it works in my fork. i'll add you as a maintainer to my fork and you can see. 

Try the link below 

https://github.com/lwasser/stravalib/compare/master...test-br?template=release-pull-request-template.md 

Then try to open a normal pr on that branch and notice when you hit "create pull request" it populates text for you in the pr text box. 

Let me know if that makes sense!

Reply:
issues are much cleaner. if the PR pull request is more effort than it's worth i can just create some template text in the release guide too that we can copy over with the check list.

Reply:
Cool, thanks for the explanation! I can see how the release template works and will not be there by default to confuse authors of regular PRs 👍.